design.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path')
```

Comment:
Since we have 2 configs here, we should add a descriptive comment on what is this particular file about.

Reply:
Will be added in next PR.